### PR TITLE
feat(layout): add patternfly to layout

### PR DESCRIPTION
**Pull request by kahboom**
tldr: This PR implements [SECURESIGN-586](https://issues.redhat.com/browse/SECURESIGN-586) to add PatternFly to the layout, leaving subcomponents alone (to follow in a separate PR shortly). Feel free to leave unmerged until the second PR with subcomponents if you do not want to merge it partially implemented (see screenshots below).

Context:
I was originally wanting to follow an approach where we can map to the upstream code by wrapping the components, such that they can support modifications, to make diffing clearer and maintenance easier when changes are inevitably made upstream, but this was taking too much time and ultimately was not as clean as I had hoped and would otherwise be over-engineered. There might also have a performance hit given that the components are not leveraging React server-components and would need to be rendered by the browser, so manually swapping out Material UI for PatternFly seems to be the best solution (for now).

Changes:

- adds PatternFly react and core to the code base
- resolves issue with Next.js + PF, where Next.js does not allow packages to import styles from their subdependencies (globally)
- adds RHTAS branding

A future PR can improve on this code by adding unit (and e2e?) tests, which upstream does not have, and possibly mapping components to sections in the upstream UI rather than 1:1 component mapping.

## Screenshots

Empty state:

Empty state

Search results:

Results

Settings:

Settings

### CODE REVIEW

**Review by Gregory-Pereira — APPROVED** [review]
No other comments
/lgtm
